goosebit/updater/manager.py

```diff
@@ -191,15 +193,22 @@ async def get_update(self) -> Tuple[HandlingType, Firmware]:
         if firmware is None:
             handling_type = HandlingType.SKIP
             self.poll_time = POLL_TIME
+            logger.info(f"Skip: no update available, device={device.uuid}")
+
         elif firmware.version == device.fw_version and not self.force_update:
             handling_type = HandlingType.SKIP
             self.poll_time = POLL_TIME
+            logger.info(f"Skip: device up-to-date, device={device.uuid}")
+
         elif device.last_state == "error" and not self.force_update:
             handling_type = HandlingType.SKIP
             self.poll_time = POLL_TIME
+            logger.info(f"Skip: device in error state, device={device.uuid}")
```

Comment:
Is this something we would consider a warning?  I'm sure later that can be extracted to be much more clear for the user on the web UI, but logging it as a warning here for now might not be a bad start?

Reply:
Agree. Changed.